ui/src/gallery/utils.ts

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable */
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
we should probably rename this to GalleryUtils in future to keep it more specific easier to quick search